baseplate/config.py

```diff
@@ -235,6 +239,25 @@ def Timespan(text):
     return datetime.timedelta(seconds=count * scale)
 
 
+def Percent(text):
+    """A percentage.
+
+    This takes a string of the form "37.2%" or "44%" and
+    returns a float in the range [0.0, 1.0].
+
+    """
+
+    if not text.endswith("%"):
+        raise ValueError("the value is not a percentage")
+
+    percentage = float(text[:-1]) / 100.
+
+    if not 0 <= percentage <= 1:
```

Comment:
🔕 

Would you want to do this validation before or after doing the float division? I don't think it'll make a material difference but it might be easier to understand if the user can't supply an argument like "100.01%" and have it work.